pokeapp/src/styling/components/_pokemon-card.scss

```diff
@@ -104,12 +104,15 @@
 .pokeball-hidden {
   position: relative;
   border-radius: 50%;
-  width: 25px;
-  height: 25px;
+  min-width: 25px;
+  min-height: 25px;
+  max-width: 25px;
+  max-height: 25px;
```

Comment:
Wouldn't you just use height and width to set the height and width rather than two settings for each?